packages/pxweb2-ui/src/lib/components/TableCard/TableCard.module.scss

```diff
@@ -0,0 +1,143 @@
+@use '../../../../style-dictionary/dist/scss/fixed-variables.scss' as fixvar;
+@use '../../../../style-dictionary/dist/css/variables.css' as var;
+@use '../../text-styles.scss';
+
+.tableCard {
+  display: flex;
+  border: 1px solid var(--px-color-border-subtle);
+  border-radius: var(--px-border-radius-large);
+  background: var(--px-color-surface-default);
+  color: var(--px-color-text-default);
+  padding: 20px;
+  position: relative;
+  cursor: pointer;
+
+  &:hover {
+    outline: 1px solid var(--px-color-border-subtle);
+
+    .title {
+      @extend .heading-underline;
+    }
+  }
+
+  &:focus-visible,
+  &:has(:focus-visible) {
+    border-radius: var(--px-border-radius-large);
+    outline: 3px solid var(--px-color-border-focus-outline);
+    outline-offset: 6px;
+    box-shadow: 0 0 0 3px var(--px-color-border-focus-boxshadow);
+  }
+}
+
+.iconWrapper {
+  min-width: 72px;
+  height: 72px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: var(--border-radius-medium, 8px);
+  margin-right: 16px;
+  svg {
+    width: 36px;
+    height: 36px;
+  }
+}
+
+.active {
+  background-color: #dbebeb;
+}
+
+.closed {
+  background-color: #ffe4e0;
+}
+
+.cardContent {
+  display: flex;
+  flex-direction: column;
+  row-gap: 16px;
+  width: 100%;
+}
+
+.titleWrapper {
+  max-width: 672px;
+
+  .title {
+    text-decoration: none;
+    color: var(--px-color-text-default);
+
+    &:focus {
+      outline: none;
+    }
+  }
+}
+
+.tableMeta {
+  display: flex;
+  flex-wrap: wrap;
+  row-gap: 8px;
+  justify-content: space-between;
+  width: 100%;
+
+  .tableId {
+    padding: 1.5px 4px 2px 4px;
```

Comment:
We have found subpixels to be problematic, check @michaelpande comments here: https://github.com/PxTools/PxWeb2/pull/616